src/Microsoft.AspNet.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -216,6 +227,40 @@ public void tcp_bind(UvTcpHandle handle, ref SockAddr addr, int flags)
         {
             handle.Validate();
             Check(_uv_tcp_bind(handle, ref addr, flags));
+            if (PlatformApis.IsWindows)
+            {
+                tcp_bind_windows_extras(handle);
```

Comment:
> What will happen if someone tries to bind to multiple network interfaces (using different SockAddrs) on the same port. Will they get an error result from WSAIoctl on the second call to tcp_bind?

No issue its per socket; also one of them would likely be not loopback, when it becomes a noop